### PR TITLE
Local dev mock api

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
To help unblock folks who have a hard time running `vets-api` locally, I added some functionality to the mock api that we run during browser tests so we can use it for local development.

## Testing done
Manually tested that it worked as expected.

## Usage
`yarn mock-api path/to/responses.js`
See the [mocker-api usage documentation](https://github.com/jaywcjlove/mocker-api#usage) for how to use the `responses.js`.

If you need to log in, go to your browser dev tools console and enter `localStorage.setItem('hasSession', true)` and refresh the page. This will then trigger a `/v0/user` call, which will then get the mocked response of a logged-in user. (Assuming you've mocked that response, of course.)

## Screenshots
N/A

## Acceptance criteria
- [x] The mock api can be run standalone, using a file for the mock responses

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/local-dev-mock-api/index.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
+const moment = require('moment');
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/local-dev-mock-api/index.js:1]: I adapted the responses in here from the `tests/disability-benefits-helpers.js`. They're only used when you run `yarn mock-api --responses src/applications/disability-benefits/all-claims/local-dev-mock-api/index.js`.

File: src/platform/testing/e2e/mockapi.js [hunk 2]
```diff
@@ -70,10 +78,7 @@ function makeMockApiRouter(opts) {
     res.status(200).json(result);
   });
 
-  // Handle CORS preflight.
-  router.options('*', cors(corsOptions));
-
-  router.all('*', cors(corsOptions), (req, res) => {
```
Comment (cvalarida) [inline src/platform/testing/e2e/mockapi.js:76]: The `mocker-api` middleware also needed this CORS configuration, so I made it global on [line 131](https://github.com/department-of-veterans-affairs/vets-website/pull/12636/files#diff-f48c0ee0e00c3041f93dd06d3c4a7129R131).

**Review by rianfowler — APPROVED** [review]
Can you document how to use this in the readme?

**Comment by cvalarida** [comment]
Ahh! Good call. I meant to do that and then completely forgot. :+1:

**Review by Mottie — APPROVED** [review]

**Pull request merged**